text/0000-plugin-package-format.md

```diff
@@ -0,0 +1,111 @@
+# Summary
+A unified way for plugins to be packaged for all platforms.
+
+# Motivation
+Currently all platforms have their own way of shipping plugins, some even being different by the way they are installed (system-wide vs local-user), which results in higher difficulty when shipping cross-platform plugins. The current behavior also prevents some much needed future features from even working correctly.
+
+# Drawbacks
+Potentially more complicated than current logic.
+
+# Design
+All plugins should be shipped and loaded in the following format:
+
+- `/<name>/data`
+    The plugins custom data, such as localized text and images.
+- `/<name>/bin/<platform>/[.../]<name>[.<ext>]`
```

Comment:
If I'm reading this correctly, only the file with the same name as the plugin's directory will be loaded directly, correct? In that case, say we're talking about obs-browser which includes a `libcef.dll`, that'd also be placed in `/<name>/bin/<platform>/[.../]` alongside `obs-browser.<ext>`, yes?

Reply:
Yes, so for your example obs-browser, it will only attempt to load obs-browser.so and ignore all other files. Those files are left up to the plugin itself to load, if at all.